#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From b44ebbdfb95e30bcd5b2f7b52cfddb088f907a54 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Wed, 17 Apr 2024 08:53:34 +0200
Subject: [PATCH] Do not require QtMultimdediaWidgets

If someone can live without the textspeechtotext plugin
this is only optional.
---
 CMakeLists.txt | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e7a030f..dbb56630 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -101,8 +101,12 @@ add_subdirectory(textgrammarcheck)
 add_subdirectory(texttranslator)
 add_subdirectory(textemoticons)
 add_subdirectory(textutils)
-find_package(Qt6 ${REQUIRED_QT_VERSION} CONFIG REQUIRED MultimediaWidgets)
-add_subdirectory(textspeechtotext)
+
+option(FORCE_DISABLE_MULTIMEDIA "Disable the Qt6MultimediaWidgets dependency - This will disable the TextSpeechToText Plugin." OFF)
+if (NOT FORCE_DISABLE_MULTIMEDIA)
+    find_package(Qt6 ${REQUIRED_QT_VERSION} CONFIG REQUIRED MultimediaWidgets)
+    add_subdirectory(textspeechtotext)
+endif()

 find_package(Qt6TextToSpeech ${QT_REQUIRED_VERSION} CONFIG)
 set_package_properties(Qt6TextToSpeech PROPERTIES DESCRIPTION
--
2.44.0
